math/ibex-lib/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           waf 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+github.setup        ibex-team ibex-lib ibex-2.8.5
```

Comment:
We don't want the version to be "ibex-2.8.5". So add a `version` line and reference it here:

```
version             2.8.5
github.setup        ibex-team ibex-lib ibex-${version}

```

Reply:
Opps, that may not work because the portgroup will override it. You might have use another variable:

```
real_version        2.8.5
github.setup        ibex-team ibex-lib ibex-${real_version}
version             ${real_version}       

```

Reply:
Done!

Reply:
`real_version` does not work (tests are failing...)

Reply:
`set        real_version 2.8.5`